### ISSUE TITLE
mmd_loss line 43

### ISSUE THREAD

**Issue by chanshing** [lyttonhao/Neural-Style-MMD#6]
Hi, in `line 43` of `mmd_loss.py`, you wrote `dot(x, x.T)`, I think it should be `dot(x.T, x)`, correct?

**Comment by rebeen** [comment]
@chanshing @lyttonhao  sorry is there any MMD loss function using Keras?
Thank you
